app/printing/printFull/testSupportFiles/expectedObjectives.js

```diff
@@ -9,9 +9,20 @@ let expectedObjectives = {
 expectedObjectives = JSON.stringify(expectedObjectives)
 expectedObjectives = JSON.parse(expectedObjectives)
 expectedObjectives.active[0].type = 'active'
+expectedObjectives.active[0].actions[0].actionText = expectedObjectives.active[0].actions[0].description
```

Comment:
I think theres a case to be made for just having another flat JSON file here instead of one that mutates an existing file; its a difficult read.

Reply:
Fair point, I'll change it.